### PR TITLE
Use XDG_CACHE_HOME as TMPDIR

### PR DESCRIPTION
**Pull request by juliushaertl**
fix for #4

### CODE REVIEW

File: org.nextcloud.Nextcloud.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
         "--share=network",
         "--share=ipc",
         "--filesystem=home:rw",
+        "--filesystem=xdg-cache",
```
Comment (TingPing) [inline org.nextcloud.Nextcloud.json]: Why are you passing this through?

Comment (juliushaertl) [reply org.nextcloud.Nextcloud.json]: My thought was to share the lock with other running nextcloud processes. Is that also the case if we don't pass the users xdg-cache through?

Comment (TingPing) [reply org.nextcloud.Nextcloud.json]: It is not needed, the lock file will be in a readable location eitherway, the only thing this does is expose other applications caches.

Comment (juliushaertl) [reply org.nextcloud.Nextcloud.json]: OK, thanks for having a look. Fixed that.

**Comment by TingPing** [comment]
bot, build

**Pull request merged**